src/index.ts

```diff
@@ -977,27 +977,55 @@ export async function getTotalDiskCapacity() {
   return -1;
 }
 
+export async function getTotalDiskCapacityNew() {
+  if (Platform.OS === 'android' || Platform.OS === 'ios' || Platform.OS === 'web') {
+    return RNDeviceInfo.getTotalDiskCapacityNew();
```

Comment:
If there is no new code for ios or web, those need to delegate to the old method don't they? I think they will crash if it runs like this